ooniprobe/Utility/DictionaryUtility.m

```diff
@@ -9,9 +9,8 @@ + (NSDictionary *)parseQueryString:(NSString *)query {
     
     for (NSString *pair in pairs) {
         NSArray *elements = [pair componentsSeparatedByString:@"="];
-        NSString *key = [[elements objectAtIndex:0] stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-        NSString *val = [[elements objectAtIndex:1] stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-        
+        NSString *key = [[elements objectAtIndex:0] stringByRemovingPercentEncoding];
+        NSString *val = [[elements objectAtIndex:1] stringByRemovingPercentEncoding];
```

Comment:
What is the rationale of this change?

Reply:
stringByReplacingPercentEscapesUsingEncoding was deprecated